### DISCUSSION TITLE
Add QC information to collate report and individual text reports

### DISCUSSION THREAD

**Discussion by harrismia** [jodyphelan/TBProfiler#205] [Ideas]
It would be great to add the qc information: "pct_reads_mapped, num_reads_mapped, median_coverage

To the collate report and to the text reports for each sample.  I believe they are currently only in the JSON files. 

 Also   total_reads, pct_reads_unmapped, and num_reads_unmapped would be helpful metrics

Thanks for considering this!

 Michael

**Comment by harrismia** [comment]
Perhaps these fields would be reported if the user specifies   --qc ?

**Reply by jodyphelan** [reply]
Sure, I can add that in.

**Reply by harrismia** [reply]
Great!  Thanks a lot!
